service/automatic_run_experiment.py

```diff
@@ -12,27 +12,29 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Determines if an experiment should be run and runs one if necessary.
-Note that this code uses a config file for experiments that is not generic.
-Thus, it only works on the official fuzzbench service. This script can be
-run manually but is intended to be run by a cronjob."""
+"""Reads experiment-requests.yaml and determines if an experiment should run and
```

Comment:
It's a bit fuzzy to me what an experiment request is. Are we going to run experiments defined in this yaml file, or there will be a default periodic experiment that re-measures everything that has changed, and these "experiment requests" are extra measurements on top of that that otherwise wouldn't run?

Reply:
We won't use change detection for the service, I implemented that but I don't really think it is viable anymore.
Basically, the current plan is for me to manually run full experiments when big changes happen and the service to run experiments when users ask for them (e.g. when they modify their fuzzer).
Do you think this needs to be clarified in the docstring, or do you think it is fuzzy because I gave you ideas about how this will work that are no longer true?